sonar-ws/src/main/java/org/sonarqube/ws/client/permission/PermissionsWsClient.java

```diff
@@ -0,0 +1,24 @@
+package org.sonarqube.ws.client.permission;
+
+import org.sonarqube.ws.WsPermissions;
+import org.sonarqube.ws.client.WsClient;
+import org.sonarqube.ws.client.WsRequest;
+
+import static org.sonarqube.ws.client.WsRequest.newGetRequest;
+
+public class PermissionsWsClient {
+  private final WsClient wsClient;
+
+  public PermissionsWsClient(WsClient wsClient) {
+    this.wsClient = wsClient;
+  }
+
+  public WsPermissions.WsGroupsResponse groups(WsGroupsRequest request) {
+    return wsClient.execute(newGetRequest("api/permissions/groups")
+        .setMediaType(WsRequest.MediaType.PROTOBUF)
```

Comment:
The protobuf media type should not needed to be repeated here as it's implicit with the second param Parser at line 22